packages/tasks_panel/src/commands/duplicate-task.ts

```diff
@@ -0,0 +1,39 @@
+import { ConfigurationTarget, Uri, window, workspace } from "vscode";
+import { ConfiguredTask } from "@sap_oss/task_contrib_types";
+import { getLogger } from "../logger/logger-wrapper";
+import { TaskTreeItem } from "../view/task-tree-item";
+import { messages } from "../i18n/messages";
+import { cleanTasks } from "../utils/ws-folder";
+import { cloneDeep, filter, find, map } from "lodash";
+
+export async function duplicateTask(treeItem: TaskTreeItem): Promise<void> {
+  try {
+    if (treeItem.command?.arguments === undefined) {
+      return;
+    }
+    const task = treeItem.command.arguments[0];
+
+    const tasksConfig = workspace.getConfiguration("tasks", Uri.file(task.__wsFolder));
```

Comment:
can you add some comments in between for better readability? why you do the cleanTasks?

Reply:
done